deployment/terraform/variables.tf

```diff
@@ -82,75 +82,6 @@ variable "private_subnet_cidrs" {
   default = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
 }
 
-variable "aws_amis" {
-  default = {
-    "eu-west-1"      = "ami-cda4fab4"
-    "eu-west-2"      = "ami-d96786be"
-    "eu-west-3"      = "ami-6b0cba16"
-    "eu-central-1"   = "ami-5e2b75b5"
-    "us-east-1"      = "ami-d464cba9"
-    "us-east-2"      = "ami-fd221298"
-    "us-west-1"      = "ami-ff0d1d9f"
-    "us-west-2"      = "ami-c05c3bb8"
-    "ca-central-1"   = "ami-cc72f4a8"
-    "ap-southeast-1" = "ami-b61633ca"
-    "ap-southeast-2" = "ami-530fc131"
-    "ap-northeast-1" = "ami-90d6c0ec"
-    "ap-northeast-2" = "ami-a1248bcf"
-    "sa-east-1"      = "ami-b090c6dc"
-    "ap-south-1"     = "ami-32c9ec5d"
-  }
-}
-
-variable "amis_20_03" {
-  default = {
-    "ap-east-1"      = "ami-0d18fdd309cdefa86"
-    "ap-northeast-1" = "ami-093d9cc49c191eb6c"
-    "ap-northeast-2" = "ami-0087df91a7b6ebd45"
-    "ap-south-1"     = "ami-0a1a6b569af04af9d"
-    "ap-southeast-1" = "ami-0dbf353e168d155f7"
-    "ap-southeast-2" = "ami-04c0f3a75f63daddd"
-    "ca-central-1"   = "ami-02365684a173255c7"
-    "eu-central-1"   = "ami-0a1a94722dcbff94c"
-    "eu-north-1"     = "ami-02699abfacbb6464b"
-    "eu-west-1"      = "ami-02c34db5766cc7013"
-    "eu-west-2"      = "ami-0e32bd8c7853883f1"
-    "eu-west-3"      = "ami-061edb1356c1d69fd"
-    "sa-east-1"      = "ami-09859378158ae971d"
-    "us-east-1"      = "ami-0c5e7760748b74e85"
-    "us-east-2"      = "ami-030296bb256764655"
-    "us-west-1"      = "ami-050be818e0266b741"
-    "us-west-2"      = "ami-06562f78dca68eda2"
-  }
-}
-
 variable "azs" {
```

Comment:
`azs`? avaiability zones? are `a` and `b` somehow given by aws or defined by us?

Reply:
both, they are a standard name used by AWS and we decide which we use, it could be `["a", "b", "c"]` for example